lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -2411,7 +2421,7 @@ DEFINE_ISXLOCALEAVAILABLE(PR, uloc)
         {
             return unum_formatDouble(*fmt, num, buf, bufLen, nullptr, status);
         }, scriptContext->GetRecycler(), &formatted, &formattedLen);
-        JavascriptOperators::InitProperty(part, PropertyIds::type, library->GetIntlLiteralPartString());
+        JavascriptOperators::InitProperty(part, PropertyIds::type, library->GetIntlUnknownPartString());
```

Comment:
>GetIntlUnknownPartString [](start = 76, length = 24)

previously a typo or?

Reply:
Nope, this was part of the Intl call for May, where we agreed that ICU part types that we don't know how to map should be treated as `{ type: "unknown" }`. Same witht he assert removal above.